week2-day2/warmup.js

```diff
@@ -0,0 +1,56 @@
+var petnames = "Tyson";
+
+var toes = 10;
+
+var tired = false;
+
+var fruits = ['banana','apple','orange'];
+
+var friends = [['Eric', 'blue'], ['Rachel', 'green'], ['Jen','gray']];
+
+var person = { name: 'Eric', toes: '10', hairColor: 'blonde'}
+	console.log("Your buddy Eric has 10 toes and his hair is blonde");
```

Comment:
Instead of _statically_ typing the value of name, toe count, and hair color, please attempt to access the values using the person variable that is storing the object using either dot notation or bracket notation.

Example: `Your buddy ${person.name}`